runtime/runtime-standalone/src/lib.rs

```diff
@@ -0,0 +1,414 @@
+use near_pool::{types::PoolIterator, TransactionPool};
+use near_primitives::{
+    account::Account,
+    errors::RuntimeError,
+    hash::CryptoHash,
+    receipt::Receipt,
+    state_record::StateRecord,
+    transaction::{ExecutionOutcome, ExecutionStatus, SignedTransaction},
+    types::AccountId,
+    types::{Balance, BlockHeight, EpochHeight, Gas, StateChangeCause},
+};
+use near_runtime_configs::RuntimeConfig;
+use near_store::{get_account, set_account, Store, Trie, TrieUpdate};
+use node_runtime::{state_viewer::TrieViewer, ApplyState, Runtime};
+
+use std::collections::HashMap;
+use std::sync::Arc;
+
+const DEFAULT_GENESIS_HEIGHT: u64 = 3;
```

Comment:
what does "default genesis height" mean?

Reply:
corrected, thx

Reply:
> You would return the receipt outcome here and the transaction outcome will be discarded. Depending on what you want, this may be fine.

Right. There is a more fine-grained API for various needs. For example, if you interested in the direct transaction outcome:

https://github.com/nearprotocol/nearcore/blob/0ae2c75608fcf907acf6ba290fead6760e8246e2/runtime/runtime-standalone/src/lib.rs#L291-L308